cross/arm-aout-binutils/Portfile

```diff
@@ -3,6 +3,9 @@
 PortSystem          1.0
 PortGroup           crossbinutils 1.0
 
+# for version >= 2.31:
+#     *** Configuration arm-unknown-aout is obsolete.
+#     *** Support has been REMOVED.
```

Comment:
Does it make sense to keep the port around?

Reply:
I am afraid I do not know.
I certainly do not use this port, but I do not use any of the cross compilers.
Version 2.30, at least, seems to build successfully.